adyen/gateway.py

```diff
@@ -60,6 +60,7 @@ class Constants:
     SESSION_VALIDITY = 'sessionValidity'
     SKIN_CODE = 'skinCode'
     SHIP_BEFORE_DATE = 'shipBeforeDate'
+    ADDITIONAL_DATA = 'additionalData.'
```

Comment:
It looks like none of the other fields have the trailing `.`. Can we avoid this inconsistency?